src/rb3keyboard.c

```diff
@@ -1,181 +1,221 @@
-//janifr
-//based on rbkit.c by ssj71
-//rbkit.c
+// janifr, jasperro
+// based on rbkit.c by ssj71
+// rbkit.c
 #include "rb3keyboard.h"
 //#include "constants.h"
 
-void init_rb3_keyboard(MIDIDRUM* MIDI_DRUM)
```

Comment:
it looks like your editor changed all the 4 space tabs to 2 space. Could you fix that too?